CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/SaveGraphPlugin.java

```diff
@@ -42,30 +46,49 @@
 public class SaveGraphPlugin extends SimplePlugin {
 
     public static final String GRAPH_PARAMETER = PluginParameter.buildId(SaveGraphPlugin.class, "graphId");
+    public static final String FILE_PATH_PARAMETER = PluginParameter.buildId(SaveGraphPlugin.class, System.getProperty("user.home"));
```

Comment:
The second argument here should be a parameter id, not a default value. You can set a default value below using `Parameter.setStringValue`.